### ISSUE TITLE
New request:

### ISSUE THREAD

**Issue by hamoudak** [openzim/zim-requests#1188]
*Please use the following format for a ZIM creation request (and delete unnecessary information)*

- Website URL: https://onelook.com/
- https://www.onelook.com/?d=wik
- License: free
- Desired ZIM Title: OneLook
- Desired ZIM Description: OneLook Dictionary Search and Thesaurus
- Desired ZIM Icon –png : https://www.onelook.com/apple-touch-icon-57x57.png
- Language (ISO 639-3): eng
- Is this a MediaWiki?: no

**Comment by benoit74** [comment]
Unfortunately due to the dynamic nature of this website (it issues a server request for every term entered in the search box), we cannot use our general purpose zimit scraper.

A custom scraper is needed which will be responsible to put the database in a ZIM, and develop a web UI capable to interact with the in-ZIM database (and query it with JS). Not something infeasible and probably not extremely hard, but definitely going to take some time, so without funding or a volunteer contributor committing to this, we might never see this happen, at least not very soon.

**Comment by benoit74** [comment]
Edit: in fact this website is only an index to existing online dictionaries. So for offline usage, it means one has to also put all dictionaries in the ZIM. Aside from being a huge work we are probably never going to be able to complete, most of the dictionnary are probably not free at all. Either I get it wrong, or this is the kind of request we cannot fulfill.

**Comment by Popolechien** [comment]
Yeah I was going to say there probably is a potential rights issue. Would suggest we close as `wontfix` at this stage.

**Comment by hamoudak** [comment]
it takes definations from just one free website; wiktionary. maybe you can exclude all other dictionaries that point to any other external dic, if it happend and there was a scrapper.  I don't know .

now I wanted to ask you if this forum can be made ; am not sure of this .
https://www.afaqattaiseer.net/  
 it's a Powered by vBulletin® Copyright ©2000 - 2024, Jelsoft Enterprises Ltd. 

the second forum on my request list is:
https://majles.alukah.net/

Powered by ®vBulletin 
Copyright © 2024 vBulletin Solutions،

**Comment by hamoudak** [comment]
yes I know, it was just the website style was good for displaying the written english and other good stuff. absolutely I have wiktionary and everything is recommended.  ok, thank you so much.
